### ISSUE TITLE
See also classes in element path

### ISSUE THREAD

**Issue by JanisE** [ckeditor/ckeditor4#2688]
## Type of report

Feature request

## Provide description of the new feature

Currently, the element path at the bottom displays only element tag names. However, if I've created custom styles by adding various classes to, say, `span` or `p` elements, there is no indication in the path, which style I'm in at the moment. As the path may look something like `body > p > span`, or `body > p` for various locations, where the `span` and `p` classes are different, but they all look at the same in the path.

**Comment by mlewand** [comment]
Currently you can already customize it using `data-cke-display-name` attribute. This is actually what Widget plugin use to provide custom names: https://github.com/ckeditor/ckeditor-dev/blob/7c937e8/plugins/widget/plugin.js#L671

But I agree that more convenient way would be to expose an event that you can listen to and customize the name of a particular element.

**Comment by JanisE** [comment]
Oh, there is that. But that attribute would be present on the final article output, too, right?

**Comment by mlewand** [comment]
@JanisE that's right, so I'd say it could be used as a workaround as the attribute would stay in your final output.

Eventually there's a workaround to that too, you can add filter that removes this when getting the data. So it all can be worked around till this feature is done.
